proto/osmosis/gamm/v1beta1/query.proto

```diff
@@ -231,6 +259,50 @@ message QuerySpotPriceResponse {
   string spot_price = 1 [ (gogoproto.moretags) = "yaml:\"spot_price\"" ];
 }
 
+//=============================== EstimateSwapExactAmountIn
+// Deprecated: use QuerySwapExactAmountInRequest in x/swaprouter.
+message QuerySwapExactAmountInRequest {
+  // TODO: CHANGE THIS TO RESERVED IN A PATCH RELEASE
+  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  uint64 pool_id = 2 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
+  string token_in = 3 [ (gogoproto.moretags) = "yaml:\"token_in\"" ];
+  repeated osmosis.swaprouter.v1beta1.SwapAmountInRoute routes = 4 [
+    (gogoproto.moretags) = "yaml:\"routes\"",
+    (gogoproto.nullable) = false
+  ];
```

Comment:
Should we set all these fields to [deprecated = true]?

Reply:
Yeah, I will be trying that in the future PR. Here, I would like to bring back the previously removed messages and queries as is.

The plan is to implement the entrypoints to route to swaprouter and try marking them as proto-deprecated in the next PR.